partner_create_by_vat/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+==============================================
+Automatic partner creation based on VAT number
+==============================================
+
+This module allows you to create the partners (companies) based on their VAT number.
+Name and address of the partner will automatically be completed via VIES Webservice.
+
+VIES Service (based on stdnum python)
+http://ec.europa.eu/taxation_customs/vies/vieshome.do
+
+Installation
+============
+
+To install this module, you need to:
+
+#. Clone the branch 10.0 of the repository https://github.com/OCA/partner-contact
+#. Add the path to this repository in your configuration (addons-path)
+#. Update the module list
+#. Search for "Partner Create by VAT" in your addons
+#. install the module
+
+Usage
+=====
+
+On the partner's form view you will have a button in the header, called
+"Get Vies Data", available only on companies (is_company field set to True).
+Clicking the button will fetch data, when available, from the VIES Webservice, for most of
+the EU countries.
+
```

Comment:
@feketemihai could you please show my an example? I'm not able to use it. I've tried with ESA09000928 but it looks module doesn't works, it only takes Country field